### PR TITLE
MOBILE-1604 update iOS SDK to 13.3.2

### PR DESCRIPTION
**Pull request by marc-scig**
This brings our titanium module into the modern age and updates the iOS SDK to 13.3.2. In order to do this cleanly I recreated the iOS module project from scratch and updated to the latest default Xcode project settings. More recent versions of appc/titanium do automatic linking of frameworks at project build time, and will package any frameworks you drop in the `platform` subdirectory into the distribution zip. The module itself is still a static lib, but this means we can use dynamic frameworks for our own dependencies.

Becuse Cocoapods/xcworkspace support doesn't appear to be fully baked yet, I ended up using Carthage here. Updating the frameworks now just requires bumping the `ios-library` version the Cartfile and running a script. The Xcode project also has the Carthage build directory in its framework search paths, so building the module directly in Xcode is straightforward, as well as on the command line.

### CODE REVIEW

File: ios/Classes/TiAirshipModule.m [hunk 1]
```diff
@@ -1,56 +1,58 @@
...
+@implementation TiAirshipModule
 
 #pragma mark Internal
 
-// this is generated for your module, please do not change it
--(id)moduleGUID {
-	return @"fbe797b5-475d-456a-bbb4-9bb9919fd63f";
+// This is generated for your module, please do not change it
+- (id)moduleGUID {
+    return @"240dc468-ccad-479d-9510-14e9a7cae5c9";
```
Comment (marc-scig) [inline ios/Classes/TiAirshipModule.m:21]: I think this was previously incorrect, set me straight if I'm wrong

File: ios/Classes/TiAirshipAutopilot.m [hunk 2]
```diff
@@ -56,24 +38,13 @@ + (void)didFinishLaunching {
...
-                        @selector(resources),
-                        method_getImplementation(swizzleMethod),
-                        method_getTypeEncoding(swizzleMethod));
-
-
     // Titanium forwards application:didReceiveRemoteNotification:fetchCompletionHandler: to
     // application:didReceiveRemoteNotification:, however that method is not defined. We will
     // add the method if missing.
     id delegate = [UIApplication sharedApplication].delegate;
     if (delegate && ![delegate respondsToSelector:@selector(application:didReceiveRemoteNotification:)]) {
```
Comment (rlepinski) [inline ios/Classes/TiAirshipAutopilot.m]: i think this is no longer needed

File: ios/Classes/TiAirshipModule.m (hunk 1 above)
Comment (rlepinski) [reply ios/Classes/TiAirshipModule.m:21]: i think its supposed to match

**Review by rlepinski — APPROVED** [review]

File: ios/Cartfile [hunk 3]
```diff
@@ -0,0 +1 @@
+github "urbanairship/ios-library" == 13.1.1
```
Comment (BrianBatchelder) [inline ios/Cartfile]: FWIW, latest release is 13.3.2

File: ios/Resources/README.md [hunk 4]
```diff
@@ -0,0 +1,10 @@
+Files in this folder are copied directory into the compiled product directory
```
Comment (BrianBatchelder) [inline ios/Resources/README.md:1]:
```suggestion
Files in this folder are copied directly into the compiled product folder
```

File: ios/Resources/README.md [hunk 5]
```diff
@@ -0,0 +1,10 @@
...
+when the iOS app is compiled:
+
+    <project-dir>/build/iphone/build/Products/<Debug-iphonesimulator|Release-iphoneos>/<app name>.app/
+
+Place your module's iOS bundles and localization files in this folder.
+
+Files in this directory are copied directly on top of whatever files are already
```
Comment (BrianBatchelder) [inline ios/Resources/README.md:8]:
```suggestion
Files in this folder are copied directly on top of whatever files are already
```

File: ios/Resources/README.md [hunk 6]
```diff
@@ -0,0 +1,10 @@
...
+in the build directory, so please be careful that your files don't clobber
```
Comment (BrianBatchelder) [inline ios/Resources/README.md:9]:
```suggestion
in the build folder, so please be careful that your files don't clobber
```

File: ios/platform/README.md [hunk 7]
```diff
@@ -1,9 +1,9 @@
-Files in this folder are copied into the iOS build directory
+Files in this folder are copied directory into the iOS build directory
```
Comment (BrianBatchelder) [inline ios/platform/README.md:1]:
```suggestion
Files in this folder are copied directly into the iOS build folder
```

File: ios/platform/README.md [hunk 8]
```diff
@@ -1,9 +1,9 @@
...
 when the iOS app is compiled:
 
     <project-dir>/build/iphone
 
-You can place files such as asset catalog files and storyboards in this
+You can place files such as asset catalog files, .framework files and storyboards in this
 directory.
```
Comment (BrianBatchelder) [inline ios/platform/README.md:7]:
```suggestion
folder.
```

File: ios/platform/README.md [hunk 9]
```diff
@@ -1,9 +1,9 @@
...
 
 Files in this directory are copied directly on top of whatever files are already
```
Comment (BrianBatchelder) [inline ios/platform/README.md:9]:
```suggestion
Files in this folder are copied directly on top of whatever files are already
```
Or however you want to clean this up. GitHub won't let me `suggest` the "directory" in line 2.

**Review by BrianBatchelder — COMMENTED** [review]
Mostly minor stuff, although I do think we should use the latest iOS SDK, i.e. 13.3.2

**Review by BrianBatchelder — APPROVED** [review]
👍

File: ios/platform/README.md (hunk 8 above)
Comment (rlepinski) [reply ios/platform/README.md:7]: this is a generated message, not going to change it

**Pull request merged**